java/java/JavaParser.g4

```diff
@@ -39,11 +39,11 @@ compilationUnit
     ;
 
 packageDeclaration
-    : annotation* PACKAGE qualifiedName ';'
+    : annotation* PACKAGE qualifiedName ';'+
```

Comment:
I don't see such thing in the [spec](https://docs.oracle.com/javase/specs/jls/se20/html/jls-19.html#jls-19-7). Consecutive semicolons usually suggest that there are empty statements, not multiple semicolons after a single statement.

Reply:
apologize for not stating the question clearly. The situation is as follows:
According to the "compileUnit" rule: "packageDeclaration? importDeclaration* typeDeclaration*"
If there are consecutive semicolons after "packageDeclaration" or in a non-last "importDeclaration", these trailing semicolons will be parsed as "typeDeclaration". As a result, the "importDeclaration" and "typeDeclaration" after the trailing semicolons will not have a chance to be correctly parsed.